docs/getting-started/tutorial-visualizing.asciidoc

```diff
@@ -53,80 +82,82 @@ each range.
 
 . Click *Update*.
 +
-Now you can see what proportion of the 1000 accounts fall into each balance
-range.
+The pie chart displays the proportion of the 1,000 accounts that fall into each of the ranges.
 +
 [role="screenshot"]
-image::images/tutorial-visualize-pie-2.png[]
+image::images/tutorial-visualize-pie-2.png[Pie chart]
 
-. Add another bucket aggregation that looks at the ages of the account
-holders.
+. Add another bucket aggregation that displays the ages of the account holders.
 
 .. In the *Buckets* pane, click *Add*, then click *Split slices*.
+
 .. From the *Sub aggregation* dropdown, select *Terms*.
-.. From the *Field* dropdown, select *age*.
 
-. Click  *Update*.
+.. From the *Field* dropdown, select *age*, then click *Update*.
 +
 The break down of the ages of the account holders are displayed
 in a ring around the balance ranges.
 +
 [role="screenshot"]
-image::images/tutorial-visualize-pie-3.png[]
+image::images/tutorial-visualize-pie-3.png[Final pie chart]
 
 . Click *Save*, then enter `Pie Example` in the *Title* field.
 
 [float]
-[[tutorial-visualize-bar]]
-=== Bar chart
+[[visualize-geographic-information]]
+=== Visualize geographic information
+
+To visualize geographic information in the log file data, use <<maps,Maps>>.
 
-Use a bar chart to look at the Shakespeare data set and compare
-the number of speaking parts in the plays.
+. Click *Create new*, then click *Maps* on the *New Visualization* window.
 
-. Click *Create visualization > Vertical Bar*, then set the source to `shakes*`.
+. To change the time, use the time filter.
+
+.. Set the *Start date* to `May 18, 2015 @ 12:00:00.000`.
```

Comment:
Might actually need to explain how to enter this date, it's not as easy to use the time picker as we think

Reply:
I agree, which is why I added the screenshot